### DISCUSSION TITLE
Formula to calculate wake when yaw angle is involved

### DISCUSSION THREAD

**Discussion by HariAB136** [NatLabRockies/floris#744] [Q&A]
Lets say i have two turbines, placed one after the other. Now wake of the 1st turbine falls on the turbine which is behind. So a formula is used to calculate air velocity for the behind turbine, basically calculation of wake velocity. Now lets say i have yawed the turbine in the front by a certain angle, now the wake also has moved a bit away from the turbine which is behind. Now how do i calculate the air velocity in this case?

**Comment by rafmudaf** [comment]
@HariAB136 are you asking how to do this in a general sense or how does FLORIS do this?

**Reply by HariAB136** [reply]
i want both if thats possible
